packages/server/Dockerfile

```diff
@@ -12,6 +12,8 @@ WORKDIR /app/basemaps
 COPY ./basemaps-landing*.tgz /app/basemaps/
 COPY ./basemaps-server*.tgz /app/basemaps/
 
+RUN npm install lerc@^4.0.4
```

Comment:
why is this needed here? if we were missing binary deps then sharp should be missing too?

Reply:
I didn't see an error related sharp when I test the server container with screenshots. Might be the deps for `cogify` in the `cli` container? But we definitely need to install `lerc` in here to make it work.